vllm/worker/model_runner.py

```diff
@@ -1205,6 +1206,15 @@ def capture_model(self, kv_caches: List[List[torch.Tensor]]) -> None:
         slot_mapping.fill_(_PAD_SLOT_ID)
         seq_lens = torch.ones(max_batch_size, dtype=torch.int32).cuda()
         block_tables = torch.from_numpy(self.graph_block_tables).cuda()
+        previous_hidden_states = None
+        if "previous_hidden_states" in inspect.signature(
+                self.model.forward).parameters:
```

Comment:
Is there a better way to do this besides dynamic inspection ?

Reply:
let's add a comment what this is for? e.g. EAGLE

Reply:
OK. I'd feel better if it were in a separate helper function so it is hidden from view, but acknowledge that it's not much better..